js/src/ui/ParityBackground/parityBackground.js

```diff
@@ -16,26 +16,17 @@
 
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
 
 class ParityBackground extends Component {
-  static contextTypes = {
-    muiTheme: PropTypes.object.isRequired
-  }
-
   static propTypes = {
+    style: PropTypes.object.isRequired,
```

Comment:
This is just to not re-compute the background from the given seed for each render (it's computed once, at least while the seed isn't changing, by the Redux `mapStateToProps` method.)